save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ProjectController.kt

```diff
@@ -83,22 +83,23 @@ class ProjectController(
             projectPermissionEvaluator.hasPermission(authentication, it, Permission.READ)
         }
 
-    @PostMapping("/not-deleted")
+    @PostMapping("/by-filters")
     @PreAuthorize("permitAll()")
     @Operation(
         method = "POST",
-        summary = "Get non-deleted projects.",
-        description = "Get non-deleted projects, available for current user.",
+        summary = "Get projects by projects filters",
+        description = "Get projects by filters, available for current user.",
```

Comment:
1. for **the** current user.
2. "_projects_ available for the current user", or "_filters_ available for the current user". Please clarify (word order matters).

Reply:
Done